src/main/java/io/github/slsa/models/Metadata.java

```diff
@@ -0,0 +1,29 @@
+package io.github.slsa.models;
+
+import java.time.Instant;
+
+/** Other properties of the build. */
+public class Metadata {
+
+  /**
+   * Identifies this particular build invocation, which can be useful for finding associated logs or
+   * other ad-hoc analysis. The exact meaning and format is defined by builder.id; by default it is
+   * treated as opaque and case-sensitive. The value SHOULD be globally unique.
+   */
+  private String buildInvocationId;
+
+  /** The timestamp of when the build started. */
+  private Instant buildStartedOn;
```

Comment:
How will this get serialized to json?  As a string? https://github.com/in-toto/attestation/blob/main/spec/field_types.md#Timestamp